cmd/fyne/internal/commands/release.go

```diff
@@ -260,21 +261,18 @@ func (r *Releaser) packageIOSRelease() error {
 	}
 
 	payload := filepath.Join(r.dir, "Payload")
-	os.Mkdir(payload, 0750)
+	_ = os.Mkdir(payload, 0750)
 	defer os.RemoveAll(payload)
 	appName := mobile.AppOutputName(r.os, r.name)
 	payloadAppDir := filepath.Join(payload, appName)
-	os.Rename(filepath.Join(r.dir, appName), payloadAppDir)
+	if err := os.Rename(filepath.Join(r.dir, appName), payloadAppDir); err != nil {
+		return err
+	}
 
-	entitlementPath := filepath.Join(r.dir, "entitlements.plist")
-	entitlements, _ := os.Create(entitlementPath)
-	entitlementData := struct{ TeamID, AppID string }{
+	if _, err := r.writeEntitlements(templates.EntitlementsDarwinMobile, struct{ TeamID, AppID string }{
 		TeamID: team,
 		AppID:  r.appID,
-	}
-	err = templates.EntitlementsDarwinMobile.Execute(entitlements, entitlementData)
-	entitlements.Close()
-	if err != nil {
+	}); err != nil {
 		return errors.New("failed to write entitlements plist template")
 	}
```

Comment:
If this should be the same as in `packageMacOSRelease`, then `cleanup` needs to be executed here, too.

Reply:
Yes it should, thanks